app/service/auth_svc.py

```diff
@@ -112,6 +112,15 @@ def __init__(self):
         except (HTTPUnauthorized, HTTPForbidden):
             raise web.HTTPFound('/login')
 
+    async def get_permissions(self, request):
+        identity_policy = request.config_dict.get('aiohttp_security_identity_policy')
+        identity = await identity_policy.identify(request)
+        if identity in self.user_map:
+            return self.user_map[identity].permissions
+        elif 'localhost:' in request.host:
```

Comment:
can you make this localhost: a variable, since we're using it a few times in this module?